src/utils/RequestPolicyUtil.re

```diff
@@ -0,0 +1,20 @@
+type requestPolicy =
```

Comment:
This looks really sweet! Looking over this API, I'm still tinkering with a few things:

1. I'm wondering if we can just use the power of default props to take away the need for `requestPolicySet` and `requestPolicyGetter`. It seems like we could have a default set in `make` function such that, if a user doesn't set this, we'll just pass it in for them.
2. I began to work on binding `requestPolicy` using `[@bs.deriving jsConverter]` with `[@bs.as]`: https://github.com/FormidableLabs/reason-urql/blob/v1/src/UrqlTypes.re#L1-L7 This construct is a special type of Bucklescript annotation specifically for string enums like this: https://bucklescript.github.io/docs/en/generate-converters-accessors#convert-between-js-string-enum-and-bs-polymorphic-variant.

Combining the above two together I'm envisioning something like the following in `UrqlQuery.re`:

```reason
let make = (~query, ~variables, ~requestPolicy=`CacheFirst, children) => {
  /* yada yada yada */
  ~props=jsProps(~query, ~variables, ~requestPolicy=requestPolicyToJs(requestPolicy)),
}
```

I _think_ that would convert whatever polymorphic variant the user passes in for `requestPolicy` into the appropriate JS string while still maintaining type validation. What do you think? The last thing I'll add is that I like the idea of this living in `UrqlTypes.re` since we'll be needing to reference it in other parts of the API like `OperationContext`.

Reply:
I didn't know you could use default props, have included it! The api seems a lot more terse than my solution 😄!